Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -0,0 +1,69 @@
+import KsApi
+import Library
+import Prelude
+import ReactiveSwift
+import Stripe
+import UIKit
+
+internal final class AddNewCardViewController: UIViewController, STPPaymentCardTextFieldDelegate {
+
+  private weak var saveButtonView: LoadingBarButtonItemView!
+  @IBOutlet private weak var cardholderNameLabel: UILabel!
+  @IBOutlet private weak var cardholderNameTextField: UITextField!
+  @IBOutlet private weak var paymentField: STPPaymentCardTextField!
+
+  internal static func instantiate() -> AddNewCardViewController {
+    return Storyboard.Settings.instantiate(AddNewCardViewController.self)
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    let cancelButton = UIBarButtonItem(title: Strings.Cancel(),
+                                       style: .plain,
+                                       target: self,
+                                       action: #selector(cancelButtonTapped))
+    cancelButton.tintColor = .ksr_green_700
+    self.navigationItem.leftBarButtonItem = cancelButton
+
+    self.saveButtonView = LoadingBarButtonItemView.instantiate()
+    self.saveButtonView.setTitle(title: Strings.Save())
+    self.saveButtonView.addTarget(self, action: #selector(saveButtonTapped))
+    let navigationBarButton = UIBarButtonItem(customView: self.saveButtonView)
+    self.navigationItem.setRightBarButton(navigationBarButton, animated: false)
+
+    self.paymentField.delegate = self
+  }
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> settingsViewControllerStyle
+
+    _ = self.cardholderNameLabel
+      |> \.text %~ { _ in Strings.Cardholder_name() }
+      |> \.textColor .~ .ksr_text_dark_grey_900
```

Comment:
Can we use `settingsTitleLabelStyle` so it's consistent with our other form fields in Settings?

Reply:
going to use `settingsSectionLabelStyle` because we want the text color to be  `.ksr_text_dark_grey_900`